### DISCUSSION TITLE
Someone can help me to solve  this error like : “error code: TOO_MANY_TASKS”  ？

### DISCUSSION THREAD

**Discussion by bigcloud2008** [apache/doris#31147] [RFC]
Hi ,
   I found this error：
OtherMsg: 2024-02-20 01:09:56:errCode = 2, detailMessage = failed to send task: errCode = 2, detailMessage = failed to submit task. **error code: TOO_MANY_TASKS**, msg: (172.30.11.31)[TOO_MANY_TASKS]ff09e217c6a14a4b-9a081dd37dcb3507_172.30.11.31
 How  can I solve ?

**Comment by yagagagaga** [comment]
You can refer to the following link
https://github.com/apache/doris/issues/24889
https://github.com/apache/doris/issues/6340
